string tag_frame_id
float32 duration
float32 DEFAULT_DURATION = 5
---
geometry_msgs/Pose pose
bool status
